### PR TITLE
fix: Fixed language on code fence in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ Currently Vector uses Github Actions to run tests. The workflows are defined in
 
 Tests are run for all changes except those that have the label:
 
-```
+```sh
```
Comment (jszwedko) [inline CONTRIBUTING.md:175]: It seems like we want `text` here? This isn't shell code.

Comment (jamtur01) [reply CONTRIBUTING.md:175]: Is `text` a language GH's markdown recognizes? I couldn't find it in the list.

Comment (jamtur01) [reply CONTRIBUTING.md:175]: Hmmm it is in Linguist. I'll update.

Comment (jszwedko) [reply CONTRIBUTING.md]: Hmm, yeah, they refer to https://github.com/github/linguist/blob/master/lib/linguist/languages.yml for the list in https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#syntax-highlighting but it isn't clear to me how those map to the the language identifier allowed for codeblock fences.

Text is listed in there though: https://github.com/github/linguist/blob/082fd09369b607b38112bfaf68640b4d44b5f328/lib/linguist/languages.yml#L5541-L5580

**Pull request merged**